trafilatura/metadata.py

```diff
@@ -160,13 +161,13 @@ def examine_meta(tree):
             if elem.get('property') == 'article:tag':
                 tags.append(content_attr)
             elif elem.get('property') in ('author', 'article:author'):
-                author = author or content_attr
+                author = content_attr if author is None else author + '; ' + content_attr if author not in content_attr and not content_attr.startswith('http') else author
```

Comment:
Hi @felipehertzer, I'm sorry but the chained conditions are hard to read and to maintain, could you please rewrite the lines 164, 170, and 191 in a more standard fashion?
Simple if/else loops with a statement per line would work exactly the same and be much easier to understand.

Reply:
Thanks for the feedback I wasn't sure which mode to use.